src/main/java/uk/gov/ida/matchingserviceadapter/saml/api/MsaTransformersFactory.java

```diff
@@ -21,6 +20,7 @@
 import uk.gov.ida.saml.core.OpenSamlXmlObjectFactory;
 import uk.gov.ida.saml.core.api.CoreTransformersFactory;
 import uk.gov.ida.saml.core.transformers.outbound.OutboundAssertionToSubjectTransformer;
+import uk.gov.ida.saml.core.transformers.outbound.ResponseToElementTransformer;
```

Comment:
This import is added but nothing else added to this file - so I'm assuming it doesn't get used?